versions.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+    }
+  }
+  required_version = ">= 0.13"
+}
```

Comment:
Since you have this here you can probably remove the terraform section in the [`main.tf`](https://github.com/UKHomeOffice/acp-tf-endpoint/blob/v1.0.1/main.tf#L22-L24) file